src/Hooks/useStore.js

```diff
@@ -0,0 +1,20 @@
+import create from 'zustand';
+import {persist} from 'zustand/middleware';
```

Comment:
I would recommend to not add `persist` right now as it can lead to problems in the future. It would be better to integrate it later on.